client/utils.py

```diff
@@ -148,6 +149,33 @@ def write_command(fdesc, command, destinations, source=0):
     time.sleep(0.3)
 
 
+def write_command_retry(fdesc, command, destinations, source=0):
+    """
+    Writes a command, retries as long as there is no answer and returns a dictionnary containing
+    a map of each board ID and its answer.
+    """
+    write_command(fdesc, command, destinations, source)
+    reader = CANDatagramReader(fdesc)
+    answers = dict()
+
+    while len(answers) < len(destinations):
```

Comment:
If the bootloader actually receives & executes the command but for some reason the answer doesn't get through, this might even be dangerous in case of a flash write command damaging the flash by too many cycles. 
In addition, let's say we're updating 20 boards at the same time, then I don't want the update to fail, just because one board hangs, but I want to finish with the 19 that didn't.